hosting/templates/hosting/calculator_form.html

```diff
@@ -71,15 +71,6 @@
                     {% endfor %}
                 </select>
             </div>
-            <div class="help-block with-errors">
```

Comment:
Could we do this in a separate PR? Does not make sense to me to something unrelated,

Reply:
Sure. Its just a duplicate block of code. Can be removed in any PR concerned with the calculator.